src/test/java/com/salesforce/dynamodbv2/mt/mappers/MTAmazonDynamoDBByAccountTest.java

```diff
@@ -131,12 +133,9 @@ public AWSCredentialsProvider getAWSCredentialsProvider(String context) {
         @Override
         public Map<String, AmazonDynamoDB> get() {
             return ImmutableMap.of("1", amazonDynamoDBClientBuilder.withCredentials(ctx1CredentialsProvider).build(),
-                                   "2", amazonDynamoDBClientBuilder.withCredentials(ctx2CredentialsProvider).build());
+                "2", amazonDynamoDBClientBuilder.withCredentials(ctx2CredentialsProvider).build());
         }
 
     }
 
-    public static final TestAccountMapper LOCAL_DYNAMO_ACCOUNT_MAPPER = new TestAccountMapper();
```

Comment:
was there any particular (good) reason that these were down here (and, if so, is there a (good!) way to rejigger things so that the code reformatter won't move these (w/o changing the code reformatter's parameters)?)

Reply:
I generally don't like to litter classes static final's but I'm not totally against it in tests.  I'm not against making it standard by moving it to the top.  I will make this change.

Reply:
(generally speaking, I figure if there's isn't a more-than-minor reason to deviate from the style guide, we should probably adhere to the style guide—but I wasn't sure if there actually *was* a more-than-minor reason to deviate in this case...)

Reply:
There isn't really a more than minor reason.  I will fix.